capa/ida/plugin/model.py

```diff
@@ -556,7 +558,7 @@ def render_capa_doc_feature(self, parent, feature, location, doc, display="-"):
 
         if feature["type"] == "regex":
             return CapaExplorerStringViewItem(
-                parent, display, location, '"%s"' % capa.features.escape_string(feature["match"])
+                parent, display, location, "\n".join(map(lambda s: '"' + capa.features.escape_string(s) + '"', feature["matches"].keys()))
```

Comment:
@mike-hunhoff this is a hack and probably completely wrong.

really, we want to add multiple rows, one for each pair `(string, location)`

`feature` looks something like:

```py
{
  "type": "regex",
  "regex": "b",
  "matches": {
    "aaabccc": [0x401000, 0x402000],
    "xxxbzzz": [0x405000]
  }
```